### PR TITLE
halmek.co.jp

### LINKED ISSUE #120796: halmek.co.jp
### Issue URL (Social Widget)
 [https://halmek.co.jp/qa/478](https://adguardteam.github.io/AnonymousRedirect/redirect.html?url=https%3A%2F%2Fhalmek.co.jp%2Fqa%2F478)

### Screenshots
<details>
  <summary>Screenshot 1</summary>

  Screenshot 1
</details>

### System configuration

Information | value
--- | ---
AdGuard product: | AdGuard/AdGuard Pro for iOS v4.4.0
Browser: | Safari
DNS filtering: | disabled
Filters: | <b>Ad Blocking:</b><br/>AdGuard Base, <br/>AdGuard Mobile Ads<br/><br/><b>Privacy:</b><br/>AdGuard Tracking Protection<br/><br/><b>Social Widgets:</b><br/>AdGuard Social Media<br/><br/><b>Annoyances:</b><br/>AdGuard Annoyances<br/><br/><b>Language-specific:</b><br/>AdGuard Japanese

### PR DESCRIPTION
**Pull request by SKEIDs**
Fix #120796

### CODE REVIEW

File: SocialFilter/sections/specific.txt [hunk 1]
```diff
@@ -1159,6 +1159,8 @@ sugoren.com,cinemacafe.net##.social-font
 newinform.com##.wrapper-share-btn
 rugbypass.com##span[onclick^="GlobalView.share"]
 viva.pl##.share-photo
+halmek.co.jp##.detail_sns_ttl
+halmek.co.jp##.detail_sns_link
```
Comment (zloyden) [inline SocialFilter/sections/specific.txt:1163]: We can do one rule
```
halmek.co.jp##[class^="detail_sns_"]
```

**Review by zloyden — CHANGES_REQUESTED** [review]

**Review by zloyden — CHANGES_REQUESTED** [review]
> [limo.media](https://github.com/AdguardTeam/AdguardFilters/pull/121018/commits/0bb5765c91efe23e25bc899b59015969a7da2cc6)

Btw, one issue - one pull request.

**Comment by SKEIDs** [comment]
> > [limo.media](https://github.com/AdguardTeam/AdguardFilters/pull/121018/commits/0bb5765c91efe23e25bc899b59015969a7da2cc6)
> 
> Btw, one issue - one pull request.

sorry , i missed

File: SocialFilter/sections/specific.txt (hunk 1 above)
Comment (SKEIDs) [reply SocialFilter/sections/specific.txt:1163]: yeah, fixed.

**Review by zloyden — APPROVED** [review]

**Pull request merged**